circle.yml

```diff
@@ -7,6 +7,9 @@ general:
 
 checkout:
   post:
+    - |
+        curl https://raw.githubusercontent.com/conda-forge/conda-forge-build-setup-feedstock/b2a07473a735ef6378be34c805a4b484b9d84579/recipe/ff_ci_pr_build.py | \
+            python - -v --ci "circle" "${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}" "${CIRCLE_BUILD_NUM}" "${CIRCLE_PR_NUMBER}"
```

Comment:
CircleCI starts up so fast that this would only really be useful at this stage if a queue has formed and jobs are waiting. That being said, we can call this script multiple times. So we could add another call right before we build.